src/Application/VersionResolver.php

```diff
@@ -38,7 +39,10 @@ public static function resolvePackageVersion(): string
         return trim($version, '"');
     }
 
-    public static function resolverReleaseDateTime(): DateTime
+    /**
+     * @return DateTime|DateTimeImmutable
+     */
+    public static function resolverReleaseDateTime(): \DateTime
```

Comment:
@Bl00D4NGEL it seems needs to be skipped as the method actually returns the exact DateTime object.

Reply:
resolved at https://github.com/rectorphp/rector-src/pull/730

Reply:
I'm not sure if this is actually "expected"
Given your fixture skip_exactly_return_datetime.php.inc
I would expect that if a function returns "just a DateTime" I would still expect the function signature (specifically the return type) to change. Maybe I'm just dumb and don't understand why this change was necessary, can you maybe explain this to be a little more so I can understand, too?

Reply:
the docblock make phpstan error